boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -43,6 +44,9 @@
 		zephyr,bt-uart = &uart3;
 #endif
 		zephyr,uart-pipe = &uart0;
+#ifdef CONFIG_NETWORKING
+		zephyr,networking = &eth;
```

Comment:
What is this used for?  seems like we should remove it.

Reply:
My understanding that this is kinda intended to select the default networking interface.

Yeah, I don't know how that works, if at all. It looks useful, but if not really implemented, could be postponed until later indeed.

Reply:
Yeah, there's no support for this in any generation scripts, and I'm not sure what it would end up doing.  So we should remove for now.